.github/workflows/upgrade.api.versions.yml

```diff
@@ -0,0 +1,31 @@
+name: Upgrade API Versions
+
+on: 
+  workflow_dispatch:
+    inputs:
+      api-version:
+        description: 'api version in the format XX e.g 58'
+        required: true
+        type: string
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v3
+      - name: Bump Version
+        run: |
+          file_list=($(find ./sfdx-source -type f -exec grep -lP "<apiVersion>.*</apiVersion>" {} \;))
+          for file in "${file_list[@]}"; do
+            echo "Processing $file"
+            sed -i "s|<apiVersion>.*</apiVersion>|<apiVersion>${{inputs.api-version}}.0</apiVersion>|g" $file
+          done
+          sed -i "s|\"sourceApiVersion\": \".*\"|\"sourceApiVersion\": \"58.0\"|g" sfdx-project.json
```

Comment:
Should the "58.0" here be the "inputs.api-version" variable?

Reply:
No :(

Reply:
Yes :(